### PR TITLE
2466: Add codeclimate

### PR DESCRIPTION
**Pull request by sarahsporck**
### Short description

Add codeclimate.

### Proposed changes

- Format test coverage & upload to codeclimate

### Side effects

none

### Resolved issues

Fixes: #2466 

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,19 +3,23 @@
...
 ## Contents
 
-- [Contributing](#contributing)
-- [Project Structure](#project-structure)
-- [Getting Started](#getting-started)
-  - [Prerequisites](#prerequisites)
-  - [Project setup](#project-setup)
-- [Further Reading](#further-reading)
+- [integreat-app](#integreat-app)
+  - [Contents](#contents)
```
Comment (steffenkleinle) [inline README.md:16]: Recursive contents? I think this should be reverted

**Review by steffenkleinle — APPROVED** [review]
Looking good!

File: README.md (hunk 1 above)
Comment (sarahsporck) [reply README.md:16]: 😅 not sure what happened here

**Review by LeandraH — APPROVED** [review]
Very cool, thank you!

**Review by ztefanie — APPROVED** [review]

**Pull request merged**